package.json

```diff
@@ -65,27 +65,30 @@
   "scripts": {
     "apidocs": "jsdoc -c jsdoc.conf.json",
     "clean": "rm -fr node_modules",
-    "clean:public": "bin/cleanPublic.js",
-    "compile": "bin/compile.js",
+    "clean:public": "scripts/cleanPublic.js",
+    "compile": "scripts/compile.js",
     "test:coverage": "nyc --reporter=lcov mocha",
     "pregenerate": "yarn compile",
-    "generate": "bin/generate.js",
+    "generate": "scripts/generate.js",
     "preghpages": "yarn generate",
-    "ghpages": "bin/ghpages.js",
+    "ghpages": "scripts/publish.js",
```

Comment:
Haven't fully tested `ghpages` yet, but I'll address it when we perform the first release after this change is merged to `master`.